sys/include/net/ng_rpl/dodag.h

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (C) 2013  INRIA.
+ * Copyright (C) 2015 Cenk Gündoğan <cnkgndgn@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup net_ng_rpl
+ * @{
+ *
+ * @file        ng_rpl_dodag.h
```

Comment:
@kb2ma welcome to RIOT!

You are right, Doxygen usually does a better job of finding the file name by itself.